samples/boards/intel_s1000_crb/i2s/prj.conf

```diff
@@ -0,0 +1,13 @@
+CONFIG_LOG=y
+
```

Comment:
No need to increase the LOG buffer here? We had crashes in the tests folder due to that.